packages/flutter/lib/src/material/ink_well.dart

```diff
@@ -332,6 +332,7 @@ class InkResponse extends StatelessWidget {
     this.onFocusChange,
     this.autofocus = false,
     this.statesController,
+    this.hoverDuration = const Duration(milliseconds: 50),
```

Comment:
Prefer to make this default to null and introduce the default with `hoverDuration ?? const Duration(milliseconds: 50)`. That will make it easier to override the default via a theme later.

Reply:
Done